apps/admin-gui/src/app/shared/components/dialogs/add-member-to-resource-dialog/add-member-to-resource-dialog.component.html

```diff
@@ -0,0 +1,98 @@
+<div class="{{theme}}">
```

Comment:
![image](https://user-images.githubusercontent.com/32871167/87855183-dba97200-c916-11ea-9847-fbd7be895fc2.png)
Height of dialog too big. Tested on 15.6 display.